platform/core/src/types/ParamsType.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * The type of generic parameter objects - this fixes issues with
+ * Object and object and any in many cases.
+ */
+type Params = Record<string, unknown>;
+export default Params;
```

Comment:
This is again another type that I don't think we need at all. `Record<string, unknown>` is well defined. What if there is like `Record<number, unknown>` or `Record<unkonwn, unknown`.